src/styles/card.scss

```
@@ -1,28 +1,39 @@
 @import "variables";
+@import "stars";
 
 .card {
   display: flex;
   flex-direction: column;
   box-sizing: border-box;
   width: 200px;
-  
+
   border: 1px solid #f3f3f3;
   border-radius: 5px;
-  
+
   padding: 15px 14px;
-  margin-bottom: 50px;
 
   &__img {
     width: 160px;
     height: 134px;
-    margin: 15px 5px 42px;
+    margin: 2px 2px 29px;
+    padding: 10px 0;
+
+    border: 3px solid transparent;
+
+    transition: 1s;
```

Вказуй чому конкретно задаєш анімацію 
```suggestion
    transition: transform 1s;
```